tests/suites/test_suite_ssl.function

```diff
@@ -63,6 +63,45 @@ exit:
 }
 #endif
 
+#if defined(MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED) && \
+    defined(MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH) && \
+    defined(MBEDTLS_SSL_PROTO_TLS1_2) && \
+    defined(PSA_WANT_ECC_SECP_R1_384) && \
+    defined(PSA_WANT_ALG_SHA_256)
```

Comment:
```suggestion
```
We should be able to just remove these guards I'd say. Did you experiment otherwise?